stubs/docutils/docutils/frontend.pyi

```diff
@@ -60,6 +62,14 @@ class OptionParser(optparse.OptionParser, SettingsSpec):
     default_error_encoding_error_handler: ClassVar[str]
     config_section: ClassVar[str]
     version_template: ClassVar[str]
+    def __init__(
+        self,
+        components: Iterable[Type[Parser]] = ...,
+        defaults: Mapping[str, object] = ...,
```

Comment:
```suggestion
        defaults: Mapping[str, Any] = ...,
```

(For the same reasons given before.)

Reply:
Fixed! And it looks like there are no more references to `object` :)